test/NuGet.Core.FuncTests/NuGet.Packaging.CrossVerify.Verify.Test/VerifySignedPackages.cs

```diff
@@ -346,6 +347,36 @@ public async Task VerifySignaturesAsync_PreGenerateSignedPackages_AuthorSigned_R
             }
         }
 
+        [Theory]
+        [MemberData(nameof(FolderForWindows_NetFulFramework))]
+        public async Task VerifySignaturesAsync_PreGenerateSignedPackages_AuthorSigned_TimeStampedWithNoSigningCertificateUsage_Throws(string dir)
+        {
+            // Arrange
+            var caseName = "ATNOCERTIFICATEUSAGE";
```

Comment:
A couple problems:
* This test naming scheme is cryptic.  It's not clear how this was generated, what it means, or how I would generate a new name successfully.
* `caseName` is confusing; don't we already have a test case name with `VerifySignaturesAsync_PreGenerateSignedPackages_AuthorSigned_TimeStampedWithNoSigningCertificateUsage_Throws`?
* Two files are expected to have the same string literal for everything to work successfully.

Reply:
Discussed few possible options with @dtivel 
- Build case names dynamically using a builder design pattern. The problem with this approach is it will be hard to search for case names during design/compile time.
- Use string constants for case names but we can't avoid duplicates values (if created unintentionally).

@dtivel  suggested to use `enum` that can help solve above drawabacks.